packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -286,16 +286,19 @@ class _TooltipOverlay extends StatelessWidget {
             opacity: animation,
             child: new Opacity(
               opacity: 0.9,
-              child: new Container(
-                decoration: new BoxDecoration(
-                  color: darkTheme.backgroundColor,
-                  borderRadius: new BorderRadius.circular(2.0),
-                ),
-                height: height,
-                padding: padding,
-                child: new Center(
-                  widthFactor: 1.0,
-                  child: new Text(message, style: darkTheme.textTheme.body1),
+              child: new ConstrainedBox(
+                constraints: new BoxConstraints(minHeight: height),
```

Comment:
you can just pass the constraints to the Container, no need to add a ConstainedBox, since you already have a Container.